lib/manageiq/loggers/base.rb

```diff
@@ -23,7 +23,7 @@ def wrap(source, *loggers)
         end
       end
 
-      def initialize(*_, **_)
+      def initialize(_logdev = nil, *_, **_)
```

Comment:
I'm not sure I understand how this is different than what was there before.  Previously we just pass along what was there.  Is this specific to a particular Ruby version?

Reply:
Logger initialize requires 1+ arguments.  Nothing is providing a logdev default previously for Base or JSONLogger.

```
  def initialize(logdev, shift_age = 0, shift_size = 1048576, level: DEBUG,
                 progname: nil, formatter: nil, datetime_format: nil,
                 binmode: false, shift_period_suffix: '%Y%m%d')
```

Reply:
I figured it was better to match the interface as `super` with a default than to explicitly pass it in the super call.

Reply:
Yeah I guess you could argue that you should be passing in something for the first arg since Logger.new also doesn't have a default value (Logger.new fails with the same error)

Reply:
```
[1] pry(main)> require "logger"
[2] pry(main)> Logger.new
~/.gem/ruby/3.3.6/gems/logger-1.6.5/lib/logger.rb:581:in `initialize': wrong number of arguments (given 0, expected 1..3) (ArgumentError)
	from (pry):2:in `new'
	from (pry):2:in `__pry__'
	from <internal:kernel>:187:in `loop'
[3] pry(main)> Logger.new(nil)
#<Logger:0x0000000133d91708 @default_formatter=#<Logger::Formatter:0x0000000133d90f88 @datetime_format=nil>, @formatter=nil, @level=0, @level_override={}, @logdev=nil, @progname=nil>
```

We're intentionally trying to match the Logger gem's interface.

Reply:
I haven't figured out the automation engine failures in 7.1 that raised this error.  I'll reopen if I figure it out.  It seemed odd we provided a default logdev for containers and journald loggers but not any of the others.